build.gradle.kts

```diff
@@ -73,38 +77,38 @@ tasks.withType<KotlinCompile> {
     }
 }
 
+@Suppress("UnstableApiUsage")
 gradlePlugin {
+    website.set("https://zerite.dev")
+    vcsUrl.set("https://github.com/HubbleMC/gradle-plugin")
+
     plugins {
-        create("gg.hubblemc.paper") {
-            id = "gg.hubblemc.paper"
+        named("gg.hubblemc.paper").configure {
```

Comment:
Should be able to do `"gg.hubblemc.paper" { ...` instead (if you want to).
Same below.